### PR TITLE
[hadoop-resource-manager] Hard-code ip to ip map in /etc/hosts file.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

**Comment by fanyangCS** [comment]
for "/etc/hosts", is this in host filesystem or that in RM's docker image?

**Comment by ydye** [comment]
> for "/etc/hosts", is this in host filesystem or that in RM's docker image?

RM's container.

File: src/hadoop-resource-manager/deploy/hadoop-resource-manager.yaml.template [hunk 1]
```diff
@@ -45,6 +45,8 @@ spec:
           name: yarn-resourcemanager-storage
         - mountPath: /var/lib/hadoopdata
           name: hadoop-tmp-storage
+        - mountPath: /host-configuration
```
Comment (fanyangCS) [inline src/hadoop-resource-manager/deploy/hadoop-resource-manager.yaml.template]: why do you this (and the configmap)?

Comment (ydye) [reply src/hadoop-resource-manager/deploy/hadoop-resource-manager.yaml.template]: One machine may have several eth and serveral ip-address. The host-configuration has a yaml containing the machine-list. We can grep the file to find the ip of the machine which configured in our configuration

**Comment by ydye** [comment]
This fix won't work, and i will try to investigate this issue. Try to fix in another way. @sterowang @fanyangCS

**Pull request closed without merging**